docs/src/gui/qtvcp-code-snippets.adoc

```diff
@@ -22,7 +26,8 @@ if self.w.PREFS_:
     self.string_value = self.w.PREFS_.getpref('String_value', 'on', str, 'CUSTOM_FORM_ENTRIES')
 ----
 
-Under the 'def closing_cleanup__(self):' function add:
+.Writing preferences at close time
+Under the *`def closing_cleanup__(self):`* function add:
```

Comment:
I know "under" is the original wording, but wouldn't "in" or "inside" fit better?

Reply:
It is ambiguous. When I read it, I thought it was meant "below the function definition". From how you read it, I would suggest "directly under the line".